src/layers.jl

```diff
@@ -91,31 +91,95 @@ end
 skip_identity(inplanes, outplanes, downsample) = skip_identity(inplanes, outplanes)
 
 """
-    addrelu(x, y)
+    mlpblock(planes, hidden_planes, dropout = 0.; dense = Dense, activation = gelu)
 
-Convenience function for `(x, y) -> @. relu(x + y)`.
-Useful as the `connection` argument for [`resnet`](#).
-See also [`reluadd`](#).
+Feedforward block used in many vision transformer-like models.
+
+# Arguments
+- `planes`: Number of dimensions in the input and output.
+- `hidden_planes`: Number of dimensions in the intermediate layer.
+- `dropout`: Dropout rate.
+- `dense`: Type of dense layer to use in the feedforward block.
+- `activation`: Activation function to use.
 """
-addrelu(x, y) = @. relu(x + y)
+function mlpblock(planes, hidden_planes, dropout = 0.; dense = Dense, activation = gelu)
```

Comment:
This still seems backwards. Two integers and a real number is a bit confusing, but integers and a function is standard elsewhere. All other `dropout`s are keywords. 
```suggestion
function mlpblock(planes::Integer, hidden_planes::Integer, activation = gelu; dense = Dense, dropout::Real = 0.0)
```
```suggestion
function mlpblock(planes, hidden_planes, dropout = 0.; dense = Dense, activation = gelu)
```
```suggestion
function mlpblock(planes, hidden_planes, dropout = 0.; dense = Dense, activation = gelu)
```
```suggestion
function mlpblock(planes, hidden_planes, dropout = 0.; dense = Dense, activation = gelu)
```
```suggestion
function mlpblock(planes, hidden_planes, dropout = 0.; dense = Dense, activation = gelu)
```
```suggestion
function mlpblock(planes, hidden_planes, dropout = 0.; dense = Dense, activation = gelu)
```

Reply:
I kept it that way because it is very unlikely that the activation function changes, while dropout is tweaked regularly in usage - but if you think it's still preferred the other way, I'll change it

Reply:
That's true. Then it makes sense for it to be a kwarg.